tests/Feature/UploadPrescriptionTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use App\Enums\SubmissionStatuses;
+use App\Models\Submission;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Http\Response;
+use Illuminate\Http\UploadedFile;
+
+uses(RefreshDatabase::class);
+
+beforeEach(function () {
+    $this->patient = User::newFactory()->patient()->patientInformation()->create();
+    $this->doctor = User::newFactory()->doctor()->create();
+    $this->otherDoctor = User::newFactory()->doctor()->create();
+    $this->pendingSubmission = Submission::newFactory()->create(['patient_id' => $this->patient->id]);
+    $this->inProgressSubmission = Submission::newFactory()->create([
+        'patient_id' => $this->patient->id,
+        'doctor_id' => $this->doctor->id,
+        'status' => SubmissionStatuses::InProgress
+    ]);
+    $this->invalidFileExtension = [
+        'prescription' => UploadedFile::fake()->create('prescription.pdf')
+    ];
+    $this->validPrescription = [
+        'prescription' => UploadedFile::fake()->create('prescription.txt')
+    ];
+});
+
+test('it throws an exception if not in progress submission', function () {
```

Comment:
Just a suggestion, but instead of having `test` and using the word 'it', you can use the function it and remove the word from the name of the function. You will still read it like `it('throws...` and when you ran the tests they will be `it throws ...`.